api/swagger.yaml

```diff
@@ -604,6 +604,49 @@ definitions:
         items:
           type: 'string'  #empty = all
 
+  ScanInterval:
+    type: object
+    properties:
+      repeatEvery:
+        type: 'integer'
+        format: 'uint32'
+      timeUnit:
+        $ref: '#/definitions/TimeUnit'
+      at:
+        type: 'string'
+        format: date-time
+
+  TimeUnit:
+    description: 'time unit of schedule scan'
+    type: 'string'
+    readOnly: true
+    enum:
+      - HOURS
+      - DAYS
+      - WEEKS
+
+  ScanTime:
+    description: 'scan time of scheduled scan'
+    type: 'string'
+    readOnly: true
+    enum:
+      - LATER
+      - REPETITIVE
+
+  RuntimeScheduleScanConfig:
+    type: object
+    properties:
+      namespaces:
+        type: 'array'
+        items:
+          type: 'string'  #empty = all
+      dockerCISBenchmarkScan:
+        type: 'boolean'
+      scanTime:
+        $ref: '#/definitions/ScanTime'
+      scanInterval:
+        $ref: '#/definitions/ScanInterval'
+
   Progress:
```

Comment:
scan time and scan type need to be added also to `Progress`

Reply:
I think that in progress we need to put the start time and in the results the end time